recipes/sdl_image/all/conanfile.py

```diff
@@ -100,29 +100,30 @@ def _configure_cmake(self):
         self._cmake.definitions["BMP"] = self.options.bmp
         self._cmake.definitions["GIF"] = self.options.gif
         self._cmake.definitions["IMAGEIO"] = self.options.get_safe("imageio")
-        self._cmake.definitions["JPG"] = self.options.jpg
+        self._cmake.definitions["JPG"] = self.options.with_libjpeg
         self._cmake.definitions["LBM"] = self.options.lbm
         self._cmake.definitions["PCX"] = self.options.pcx
-        self._cmake.definitions["PNG"] = self.options.png
+        self._cmake.definitions["PNG"] = self.options.with_libpng
         self._cmake.definitions["PNM"] = self.options.pnm
         self._cmake.definitions["SVG"] = self.options.svg
         self._cmake.definitions["TGA"] = self.options.tga
-        self._cmake.definitions["TIF"] = self.options.tif
-        self._cmake.definitions["WEBP"] = self.options.webp
+        self._cmake.definitions["TIF"] = self.options.with_libtiff
+        self._cmake.definitions["WEBP"] = self.options.with_libwebp
         self._cmake.definitions["XCF"] = self.options.xcf
         self._cmake.definitions["XPM"] = self.options.xpm
         self._cmake.definitions["XV"] = self.options.xv
         # TODO: https://github.com/bincrafters/community/pull/1317#pullrequestreview-584847138
-        self._cmake.definitions["TIF_DYNAMIC"] = self.options["libtiff"].shared if self.options.tif else False
-        self._cmake.definitions["JPG_DYNAMIC"] = self.options["libjpeg"].shared if self.options.jpg else False
-        self._cmake.definitions["PNG_DYNAMIC"] = self.options["libpng"].shared if self.options.png else False
-        self._cmake.definitions["WEBP_DYNAMIC"] = self.options["libwebp"].shared if self.options.webp else False
+        self._cmake.definitions["TIF_DYNAMIC"] = self.options["libtiff"].shared if self.options.with_libtiff else False
+        self._cmake.definitions["JPG_DYNAMIC"] = self.options["libjpeg"].shared if self.options.with_libjpeg else False
+        self._cmake.definitions["PNG_DYNAMIC"] = self.options["libpng"].shared if self.options.with_libpng else False
+        self._cmake.definitions["WEBP_DYNAMIC"] = self.options["libwebp"].shared if self.options.with_libwebp else False
         self._cmake.definitions["SDL_IS_SHARED"] = self.options["sdl"].shared
 
         self._cmake.configure(build_dir="build")
         return self._cmake
 
     def build(self):
+        tools.rmdir(os.path.join(self._source_subfolder, "external"))
```

Comment:
sdl_image uses a vendored header-only `minz.h`.
I tried replacing it with our `miniz/2.2.0`.
But it looks like the API changed between 1.x and 2.x.